src/cdk-experimental/scrolling/virtual-scroll.md

```diff
@@ -0,0 +1,139 @@
+**Warning: this component is still experimental. It may have bugs and the API may change at any
+time**
+
+`<cdk-virtual-scroll-viewport>` displays large lists of elements performantly by only
```

Comment:
I think we need to explain virtual scroll a little bit. I don't think people understand the difference between virtual scroll and infinite scroll. A idea gist.

> Virtual scrolling enables developers to render large lists of data in a performant way. Loading hundreds of elements can be slow in any browser, virtual scrolling enables a performant want to simulate all items are rendered by making the height of the container element the total number of elements to be rendered and then only rendering the items in view. Virtual scroll is different from strategies like infinite scroll where it renders a set amount of elements and then when you hit the bottom renders the rest.

Reply:
made some slight tweaks and merged with my sentence above